dt-mapping/drill-down.js

```diff
@@ -24,7 +24,7 @@ let rebuild_drill_down = ( response, bindFunction, grid_id )=>{
 
             // create button
             html += `<li><button id="${window.lodash.escape( section.parent )}" type="button" ${disabled ? "disabled" : ""}
-                onclick="DRILLDOWN.get_drill_down( '${window.lodash.escape( bindFunction )}', '${window.lodash.escape( section.selected )}' )"
+                onclick="DRILLDOWN.get_drill_down( '${window.lodash.escape( bindFunction )}', '${window.lodash.escape( section.selected )}', false )"
```

Comment:
@prykon this looks like the same area as our previous conversation.

Reply:
@corsacca
We should leave the 'false' cache parameter... explanation below

**Behavior with the 'false' cache parameter:**
1. Enter a country
2. Change its name
3. Click the 'World' button in the breadcrumb
4. You're back in the main locations list with the updated country name

**Behavior without the 'false' cache parameter:**
1. Enter a country
2. Change its name
3. Click the 'World' button in the breadcrumb
4. You're back in the main locations list with not-yet updated country name

Reply:
Since this code is used in other places in the theme (metrics), how about invalidating the cache when a record is updated or deleted instead of completely disabling it?
could be as simple as doing: `window.drilldownModule.drilldown = null`